### DISCUSSION TITLE
Changing Editor Font-Family

### DISCUSSION THREAD

**Discussion by DRichter-r** [remirror/remirror#1617] [Q&A] [answered]
Hi! 
I´m trying to change the Font-Family of the editor through the ThemeProvider like so: 
![image](https://user-images.githubusercontent.com/102805001/165103967-ddcf4c02-191a-4597-94a3-eda93d2d0785.png)

This does not seem to work. Is this a bug or am i going at this the wrong way? Does anyobdy have some info on how to achieve this?

**Comment by ocavue** [comment, marked as answer]
Use `,` instead of `;`

```tsx
    <ThemeProvider theme={{ fontFamily: { default: 'cursive, "Gill Sans", Georgia' } }}>
      <Remirror manager={manager} initialContent={state} autoFocus>
        <EditorComponent />
        <EmojiPopupComponent />
      </Remirror>
    </ThemeProvider>
```

**Reply by DRichter-r** [reply]
Thanks, that worked.
